tests/integration/components/select-light-test.js

```diff
@@ -153,8 +153,9 @@ module('Integration | Component | select-light', function(hooks) {
       <SelectLight
         @options={{this.options}}
         @value={{this.value}}
-        @valueKey="val"
-        @displayKey="description" />
+        @valueKey="val">
```

Comment:
This test no longer passes in a `@displayKey` so the test description could be adjusted to reflect this. Something like `should render a named option block with a customized value key when passed array of objects` perhaps.

It could then be good to add another test that uses a custom passed in `@displayKey`